tests/helpers.py

```diff
@@ -178,6 +178,8 @@ def reset_poetry(self) -> None:
 
 
 class TestLocker(Locker):
+    __test__ = False
```

Comment:
What is this for?

Reply:
else pytest thinks that this class is defining testcases

Reply:
Could you add a short comment explaining that to avoid similar confusion in the future?

Reply:
sure